packages/client-config/API.md

```diff
@@ -26,6 +26,8 @@ export enum ClientConfigFormat {
     // (undocumented)
     JSON = "json",
     // (undocumented)
+    JSON_MOBILE = "json-mobile",

```

Comment:
https://app.asana.com/0/1205484060153101/1205931466811581/f

The other option was to add additional dimension like `--target` but that was rejected.

This should go away if/when client config is unified here https://app.asana.com/0/1205484060153101/1205931466811588/f